src/components/List.js

```diff
@@ -13,9 +21,30 @@ const List = ({ token, results }) => {
           </NavLink>
         </>
       ) : null}
-      <ul style={{ color: 'black' }}>
+      <ul>
         {results.map(result => {
-          return <li key={result.id}>{result.name}</li>;
+          const time = Math.max(...result.purchaseDates); //pulls most recent purchase date
+          return (
+            <li
+              key={result.id}
+              className={checkTime(time) ? 'deactivated' : null}
+            >
+              <label htmlFor={result.id} className="sr-only">
+                {result.name}
+              </label>
+              <input
+                type="checkbox"
+                disabled={checkTime(time)}
+                defaultChecked={checkTime(time)}
+                name={result.id}
+                id=""
```

Comment:
I think the id here needs to be `id={result.id}` for accessibility purposes, to match it with it's label

Reply:
Great catch!